src/main/scala/dpla/ingestion3/mappers/providers/NaraExtractor.scala

```diff
@@ -0,0 +1,256 @@
+package dpla.ingestion3.mappers.providers
+
+import java.net.URI
+
+import dpla.ingestion3.enrichments.{DcmiTypeMapper, VocabEnforcer}
+import dpla.ingestion3.mappers.rdf.DCMIType
+import dpla.ingestion3.mappers.xml.XmlExtractionUtils
+import dpla.ingestion3.model._
+import org.eclipse.rdf4j.model.IRI
+
+import scala.xml.{Node, NodeSeq, XML}
+
+class NaraExtractor(rawData:String) extends Extractor with XmlExtractionUtils with Serializable {
+
+  implicit val xml: NodeSeq = XML.loadString(rawData)
```

Comment:
Could you comment on why this `implicit val xml` is an implicit val?

I see that `dpla.ingestion3.mappers.xml.XmlExtractionUtils` has the methods `extractString` and `extractStrings`, which each have an `implicit val xml`, but I have the sense I'm missing something when I read this.